earth2mip/time_loop.py

```diff
@@ -60,10 +60,11 @@ def __call__(
         Args:
             x: an initial condition. has shape (B, n_history_levels,
                 len(in_channel_names), Y, X).  (Y, X) should be consistent with
-                ``grid``. ``x[:, -i]`` is the data correspond to
-                ``time - (i-1) * self.history_time_step``. Note this means that
-                ``time`` corresponds to ``x[:, -1]``...not ``x[:, 0]``.
-            time: the datetime to start with
+                ``grid``. The history dimension is in increasing order, so the
+                current state corresponds to x[:, -1].  Specifically, ``x[:,
+                -i]`` is the data correspond to ``time - (i-1) *
+                self.history_time_step``.
+            time: the datetime to start with, by default assumed to be in UTC.
```

Comment:
Is the UTC assumption requirement?

Reply:
the tzinfo can be overrided, but i realized when working on https://github.com/NVIDIA/modulus/pull/194 that we do need to document this detail. Without specifying the timezone python's behavior for computing time differences/timesteps seems system dependent.